core/templates/dev/head/domain/topic/topic-creation-backend-api.service.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about creation of topic and obtain
+ * topic_id.
+ */
+
+angular.module('oppia').factory('TopicCreationBackendApiService', [
```

Comment:
Can you please create this service in Angular? (Sorry for asking this now, I've added a note in the issue thread as well.)

You can check [this doc](https://docs.google.com/document/d/1jzwCMoZDGw4U5711mECHIsWnxDM6VMSkD6ECpPKNp70/edit?ts=5e41a770#heading=h.siedvmt9vfi), I think it has all the details on how to create a new backend-ap.service file in Angular.

Reply:
In the newest commit, the files are migrated to angular
Could you please review it
Thanks